### PR TITLE
Began working on content for Synchronization

### PR DESCRIPTION
**Pull request by Jesseadriano10**
**- What I did**
- Finished Synchronization definition and polished the content on what shared keys are.
**- How I did it**
- Transferred material from google doc into markdown file as well as info from the articles. 
**- How to verify it**
- See synchronization.md, public-and-private-keys index.md
**- Description for the changelog**

### CODE REVIEW

File: content/docs/reference/synchronization.md [hunk 1]
```diff
@@ -9,8 +9,22 @@ description: | # SEO Description of the page (Shows in google and atsign.dev sea
...
 draft: false # Change this to "true" to hide the page
-toc: false # Change this to "true" to show the table of contents
+toc: true # Change this to "true" to show the table of contents
 weight: 210 # For single pages, lower is first.
 ---
 
-TODO sync
+## Definition  
+
+Here at the @protocol, your data is encrypted with your private key and stored on your device. Periodically, this data is copied securely over to a dedicated cloud server which only you can decrypt and read since you are the only one who owns the private key. No one else, including **AtSign** can read your data. This process is known as synchronization.
```
Comment (xavierchanth) [inline content/docs/reference/synchronization.md]: "Here at Atsign" or "On the @‎protocol" for the opening to line 18.

Comment (xavierchanth) [reply content/docs/reference/synchronization.md]: "AtSign" should be lowercase "s" : "Atsign"

File: content/docs/reference/synchronization.md [hunk 2]
```diff
@@ -9,8 +9,22 @@ description: | # SEO Description of the page (Shows in google and atsign.dev sea
...
+
+## How it works with the @protocol  
+
+First we need to touch up on what local & remote secondaries are:
+1. **Remote Secondary**: A microservice with a DNS address which makes it addressable, provides a backup of your data in case of loss on the phone, provides a joining place to synchronize data if you have more than one device.
+2. **Local Secondary**: A copy of data that allows for fast interactivity and offline support. Every activated @sign gets their own remote secondary which runs on AtSign's servers. All of your data is signed and [encrypted](/docs/reference/encryption), so AtSign cannot see it.
```
Comment (xavierchanth) [inline content/docs/reference/synchronization.md]: Same thing with the company name on line 24

File: content/docs/reference/synchronization.md (hunk 1 above)
Comment (xavierchanth) [reply content/docs/reference/synchronization.md]: Change "private key" to "self encryption key"

File: content/docs/reference/synchronization.md (hunk 2 above)
Comment (xavierchanth) [reply content/docs/reference/synchronization.md]: encrypted link should be attached to the first reference of encryption in the document.

File: content/docs/reference/public-private-keys/index.md [hunk 3]
```diff
@@ -65,8 +65,15 @@ A key in the @protocol can be formed by using any alphanumeric and special chara
     
     > Note: Above Key should be part of scan verb result for only @alice and @bob 
 
-    > The owner of the secondary should be allowed to update or delete the value of a user key.
- 
+    > The owner of the secondary should be allowed to update or delete the value of a user key.  
+
+    **More context on shared keys**: We know that AtSign uses *[AES-256 & RSA-2048](docs/reference/encryption)* for symmetric and asymmetric encryption respectfully.
```
Comment (xavierchanth) [inline content/docs/reference/public-private-keys/index.md]: Same issue with company name as listed below

Comment (xavierchanth) [reply content/docs/reference/public-private-keys/index.md]: Remove "We know that"

File: content/docs/reference/public-private-keys/index.md [hunk 4]
```diff
@@ -65,8 +65,15 @@ A key in the @protocol can be formed by using any alphanumeric and special chara
...
- 
+    > The owner of the secondary should be allowed to update or delete the value of a user key.  
+
+    **More context on shared keys**: We know that AtSign uses *[AES-256 & RSA-2048](docs/reference/encryption)* for symmetric and asymmetric encryption respectfully.  
+    
+    * RSA-2048 can encrypt up to 214 bytes which does not constitute for a lot of data. However, because it is asymmetric, we can use it to ensure that you are you by verifying that you have your private key.
+
+    * AES-256 can encrypt around a whopping 250 million terabytes which is practically unlimited. The downside is that because this is a symmetric key, we cannot use it to authorize that you are you since this is shared with the recipient.
+
+    To remediate this, we can generate an AES(shared) key, and encrypt it using your public RSA key. Then, we can decrypt said AES key using your private RSA key. Since you are the only holder of your private key, I can ensure that only the two of us hold the AES key. Now we can share information back-and-forth safely using the AES as a means of encryption/decryption.
```
Comment (xavierchanth) [inline content/docs/reference/public-private-keys/index.md]: change "we" to "I" in sentence 1.
Remove comma after then sentence 2.
"key" after "AES" in last sentence.

**Review by xavierchanth — DISMISSED** [review]

**Comment by xavierchanth** [comment]
Remember to write as if you were speaking as the company.

File: content/docs/reference/synchronization.md (hunk 1 above)
Comment (JeremyTubongbanua) [reply content/docs/reference/synchronization.md]: I think the first sentence of `##Definition` should simply state what it is. (Remember that people don't like reading)
So maybe start off with `Synchronization is...`

File: content/docs/reference/synchronization.md (hunk 2 above)
Comment (Jesseadriano10) [reply content/docs/reference/synchronization.md]: I added to the first reference which occurs on the first sentence.

**Comment by Jesseadriano10** [comment]
Please see https://github.com/atsign-foundation/atsign.dev-3.0/pull/62/commits/ce6b431d5e52655b4604c22cecbb6e7f14cba942 for changes applied! Let me know if I got everything correct!

**Comment by Jesseadriano10** [comment]
> Change `Atsign.dev` -> `atsign.dev`

Where exactly is this line? I am having trouble finding it

**Comment by Jesseadriano10** [comment]
> Maybe change title to `@platform` ? or `How It Works At Atsign`

Good catch! Better to be consistent

**Comment by JeremyTubongbanua** [comment]
> > Change `Atsign.dev` -> `atsign.dev`
> 
> Where exactly is this line? I am having trouble finding it

It's a comment

**Review by JeremyTubongbanua — APPROVED** [review]
LGTM

**Pull request merged**